projects/person-search-index-from-delius/deploy/values.yaml

```
@@ -0,0 +1,46 @@
+generic-service:
+  nameOverride: person-search-index-from-delius
+  replicaCount: 1
+  resources:
+    limits:
+     cpu: 2
+     memory: 2GB
+    requests:
+    cpu: 1
+    memory: 1GB
```

```suggestion
  resources:
    limits:
      cpu: 2
      memory: 2GB
    requests:
      cpu: 1
      memory: 1GB
```
This should fix the indentation